configs/spmd/large_scale/kmeans_wide_weak.json

```
@@ -5,15 +5,18 @@
             "algorithm": {
                 "estimator": "KMeans",
                 "estimator_params": {
-                    "algorithm": "lloyd"
+                    "algorithm": "lloyd",
+		    "max_iter": 20,
+		    "n_clusters": 10,
+		    "random_state": 42
                 },
-                "estimator_methods": { "training": "fit", "inference": "predict" }
+                "estimator_methods": { "training": "fit", "inference": "" },
```

```suggestion
                "estimator_methods": { "training": "fit", "inference": "predict" },
```
For final config we can have predict (unless there are concerns with running it)